platforms/ios/Emojibase/Resources/emojibase.json

```diff
@@ -0,0 +1 @@
m-dark skin tone, medium-light skin tone","order":2656,"unicode":"🧑🏾‍🤝‍🧑🏼"},{"group":1,"hexcode":"1F9D1-1F3FE-200D-1F91D-200D-1F9D1-1F3FD","label":"people holding hands: medium-dark skin tone, medium skin tone","order":2657,"unicode":"🧑🏾‍🤝‍🧑🏽"},{"group":1,"hexcode":"1F9D1-1F3FE-200D-1F91D-200D-1F9D1-1F3FE","label":"people holding hands: medium-dark skin tone","order":2658,"unicode":"🧑🏾‍🤝‍🧑🏾"},{"group":1,"hexcode":"1F9D1-1F3FE-200D-1F91D-200D-1F9D1-1F3FF","label":"people holding hands: medium-dark skin tone, dark skin tone","order":2659,"unicode":"🧑🏾‍🤝‍🧑🏿"},{"group":1,"hexcode":"1F9D1-1F3FF-200D-1F91D-200D-1F9D1-1F3FB","label":"people holding hands: dark skin tone, light skin tone","order":2660,"unicode":"🧑🏿‍🤝‍🧑🏻"},{"group":1,"hexcode":"1F9D1-1F3FF-200D-1F91D-200D-1F9D1-1F3FC","label":"people holding hands: dark skin tone, medium-light skin tone","order":2661,"unicode":"🧑🏿‍🤝‍🧑🏼"},{"group":1,"hexcode":"1F9D1-1F3FF-200D-1F91D-200D-1F9D1-1F3FD","label":"people holding hands: dark skin tone, medium skin tone","order":2662,"unicode":"🧑🏿‍🤝‍🧑🏽"},{"group":1,"hexcode":"1F9D1-1F3FF-200D-1F91D-200D-1F9D1-1F3FE","label":"people holding hands: dark skin tone, medium-dark skin tone","order":2663,"unicode":"🧑🏿‍🤝‍🧑🏾"},{"group":1,"hexcode":"1F9D1-1F3FF-200D-1F91D-200D-1F9D1-1F3FF","label":"people holding hands: dark skin tone","order":2664,"unicode":"🧑🏿‍🤝‍🧑🏿"}],"shortcodes":["people_holding_hands"]},{"group":1,"hexcode":"1F46D","label":"women holding hands","order":2665,"tags":["couple","hand","holding hands","women"],"unicode":"👭","skins":[{"group":1,"hexcode":"1F46D-1F3FB","label":"women holding hands: light skin tone","order":2666,"unicode":"👭🏻"},{"group":1,"hexcode":"1F46D-1F3FC","label":"women holding hands: medium-light skin tone","order":2672,"unicode":"👭🏼"},{"group":1,"hexcode":"1F46D-1F3FD","label":"women holding hands: medium skin tone","order":2678,"unicode":"👭🏽"},{"group":1,"hexcode":"1F46D-1F3FE","label":"women holding hands: medium-dark skin tone","order":2684,"unicode":"👭🏾"},{"group":1,"hexcode":"1F46D-1F3FF","label":"women holding hands: dark skin tone","order":2690,"unicode":"👭🏿"},{"group":1,"hexcode":"1F469-1F3FB-200D-1F91D-200D-1F469-1F3FC","label":"women holding hands: light skin tone, medium-light skin tone","order":2667,"unicode":"👩🏻‍🤝‍👩🏼"},{"group":1,"hexcode":"1F469-1F3FB-200D-1F91D-200D-1F469-1F3FD","label":"women holding hands: light skin tone, medium skin tone","order":2668,"unicode":"👩🏻‍🤝‍👩🏽"},{"group":1,"hexcode":"1F469-1F3FB-200D-1F91D-200D-1F469-1F3FE","label":"women holding hands: light skin tone, medium-dark skin tone","order":2669,"unicode":"👩🏻‍🤝‍👩🏾"},{"group":1,"hexcode":"1F469-1F3FB-200D-1F91D-200D-1F469-1F3FF","label":"women holding hands: light skin tone, dark skin tone","order":2670,"unicode":"👩🏻‍🤝‍👩🏿"},{"group":1,"hexcode":"1F469-1F3FC-200D-1F91D-200D-1F469-1F3FB","label":"women holding hands: medium-light skin tone, light skin tone","order":2671,"unicode":"👩🏼‍🤝‍👩🏻"},{"group":1,"hexcode":"1F469-1F3FC-200D-1F91D-200D-1F469-1F
```

Comment:
Keeping this in the repo isn't ideal, is this needed for swift packages to function? Can we not build it and deploy it into a package repo ala npm for swift?

Reply:
SwiftPM deals directly with git repos, so the layout here would be common place. We could write a script to compile it to an xcframework that gets uploaded to the release binary but it's more common to ship the source with your package. Any thoughts @stefanceriu?

Reply:
Another option might be using a package plugin instead but I don't think either of them are worth the effort, writing or maintaining.